src/Clue/GraphComposer/Command/Show.php

```diff
@@ -17,13 +17,14 @@ protected function configure()
              ->setDescription('Show dependency graph image for given project directory')
              ->addArgument('dir', InputArgument::OPTIONAL, 'Path to project directory to scan', '.')
              ->addOption('format', null, InputOption::VALUE_REQUIRED, 'Image format (svg, png, jpeg)', 'svg')
-           /*->addOption('dev', null, InputOption::VALUE_NONE, 'If set, Whether require-dev dependencies should be shown') */;
+             ->addOption('no-dev', null, InputOption::VALUE_NONE, 'Hide dev dependencies.');
```

Comment:
Care to share some background, what's your motivation for hiding dev dependencies by default?

Here's my list why I think a `--no-dev` option makes more sense:

* It's in line with how composer works nowadays
* Without any options, you see all dependencies; optional arguments can be used to hide, so nothing ever goes unnoticed; this can be confusing, which options need to be passed to show everything?
* It's in line with how the other future options such as `--depth` or `--purge` are going to work
* Changing the default display options should be considered a BC break